ansible/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
   with_items: build_ossec_deb_pkg_dependencies
 
 - name: extract OSSEC archive to /tmp
-  shell: tar -xvf {{ build_path }}/{{ download_name }} -C /tmp
+  shell: tar -xvf {{ build_path }}/ossec-hids-{{ download_name }} -C /tmp
```

Comment:
The new download url is: `https://github.com/ossec/ossec-hids/archive/2.8.2.tar.gz` The downloaded file is `ossec-hids-2.8.2.tar.gz`

Think this will be getting re-org'd by @conorsch at some point in the future.